src/promptflow/tests/sdk_cli_test/e2etests/test_cli_perf.py

```diff
@@ -57,7 +57,7 @@ def subprocess_run_cli_command(cmd, time_limit=3600):
 @pytest.mark.usefixtures("use_secrets_config_file", "setup_local_connection")
 @pytest.mark.perf_monitor_test
 class TestCliPerf:
-    def test_pf_run_create(self, time_limit=35) -> None:
+    def test_pf_run_create(self, time_limit=20) -> None:
```

Comment:
What's the average time for this command? Maybe update in PR description?

Reply:
The average time is unreliable, but PC99 is quite high:

"**pfazure. run. create**" takes about 26 seconds, which should be related to me not setting the replay mode.

There are several other commands that have a maximum time consumption of over 20 seconds, even 330+, which is quite strange. I cannot reproduce it, so I need to merge this PR first, ensure the replay mode, and then check the time consumption.

Reply:
Due to data confidentiality, I won't post detailed information here.